galaxy_importer/ansible_test/job_template.yaml

```diff
@@ -15,7 +15,7 @@ spec:
         image: {image}
         resources:
           limits:
```

Comment:
Please add `requests:` to resource section.

Reply:
@cutwater I will probably set the request at:
```
requests:
  cpu: 250m
  memory: 256Mi
```

Reply:
Had to adjust to `500m` and `512Mi` accommodate larger collections